zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/SchemaManager.java

```diff
@@ -123,20 +140,35 @@ private IndexSettings getIndexSettings(final String indexName) {
 
   private Map<IndexDescriptor, Collection<IndexMappingProperty>> validateIndices(
       final IndexSchemaValidator schemaValidator, final SearchEngineClient searchEngineClient) {
-    final var currentIndices =
-        searchEngineClient.getMappings(config.getIndex().getPrefix() + "*", MappingSource.INDEX);
+    if (indexDescriptors.isEmpty()) {
+      return Map.of();
+    }
+
+    final var currentIndices = searchEngineClient.getMappings(alLIndexNames(), MappingSource.INDEX);
 
     return schemaValidator.validateIndexMappings(currentIndices, indexDescriptors);
   }
 
   private Map<IndexDescriptor, Collection<IndexMappingProperty>> validateIndexTemplates(
       final IndexSchemaValidator schemaValidator, final SearchEngineClient searchEngineClient) {
-    final var currentTemplates = searchEngineClient.getMappings("*", MappingSource.INDEX_TEMPLATE);
+    if (indexTemplateDescriptors.isEmpty()) {
+      return Map.of();
+    }
+
+    final var currentTemplates =
+        searchEngineClient.getMappings(
+            config.getIndex().getPrefix() + "*", MappingSource.INDEX_TEMPLATE);
 
     return schemaValidator.validateIndexMappings(
         currentTemplates,
         indexTemplateDescriptors.stream()
             .map(IndexDescriptor.class::cast)
             .collect(Collectors.toSet()));
   }
+
+  private String alLIndexNames() {
```

Comment:
🔧 I guess this is a typo?
```suggestion
  private String allIndexNames() {
```